cmd/fulcio/createcerts/main.go

```diff
@@ -60,6 +62,7 @@ func main() {
 
 	versionInfo := version.GetVersionInfo()
 	logging.FromContext(ctx).Infof("running create_certs Version: %s GitCommit: %s BuildDate: %s", versionInfo.GitVersion, versionInfo.GitCommit, versionInfo.BuildDate)
+	logging.FromContext(ctx).Warn("DEPRECATED: create certificates externally instead of relying on this command")
```

Comment:
is this sufficient to get people to move over or could we force them use use a `--i-know-this-is-deprecated` flag?

Reply:
It's kind of a noop because no one is running this by hand, it's a kubernetes Job run by the fulcio helm chart. It will be more important to deprecate having it enabled in the chart.